index.html

```diff
@@ -360,6 +360,37 @@ <h2>Skills</h2>
             </ul>
         </div>
     </section>
+    <!-- End Section part -->
+
+    <!-- Start Contact form -->
+
+    <div class="contact-form">
+        <div class="formbg" alt="form background"></div>
+        <div class="form-header">
+          <div class="form-header__title">
+                <h4>Contact me</h4>
+            </div>
+            <div class="form-header__text">
+                <p>If you have an application you are interested in developing, a feature that you need built or a project that needs coding. 
+                I’d love to help with it!</p>
+         </div>
+        </div>
+        <div class="inputs">
+            <form action="" class="form">
+              <input type="text" class="item1" name="name" id="name" placeholder="Lorem Ipsum" required maxlength="30" >
+              <input type="email" class="item1" name="email" id="email" placeholder="loramipsum@gmail.com" required pattern=".+@beststartupever.com">
+              <textarea name="message" id="message" class="item3" placeholder="Write your message here" maxlength="500"></textarea>
+              <button class="formbtn" type="submit">Get in touch</button>
```

Comment:
Kindly fix your submit button's styling for hover and active state like on the Figma design guideline. We need to stick to the design as much as possible.
<img width="455" alt="Screenshot 2023-04-04 at 11 40 21 PM" src="https://user-images.githubusercontent.com/25887253/229915613-7752ae14-54f7-4baa-881c-421f8aea6b93.png">

Reply:
@huseyinbiyik 
how can I access to the submit button's styling for hover and active state in Figma, because it does not show the style for hover and active